packages/core/models/translation-object.model.ts

```diff
@@ -41,6 +41,11 @@ export class TranslationObject {
         return { ...values };
     }
 
+    public isTranslatedIn(languages: string[]): boolean {
+        const translatedLanguages = Object.keys(this.getAll());
+        return languages.every(language => translatedLanguages.includes(language));
+    }
```

Comment:
I think that `every` of an empty array will return true, wich is not the expected behaviour. You should test that edge case

Reply:
Indeed